packages/eslint-config/README.md

```diff
@@ -0,0 +1,41 @@
+# @loopback/eslint-config
+
+Shared ESLint config to enforce a consistent code style for LoopBack development
+
+## Installation
+
+```shell
+$ npm install --save @loopback/eslint-config
+```
+
+## Basic Use
+
+An example `.eslintrc.json` file:
+
+```json
+{
+  "extends": "@loopback/eslint-config"
+}
+```
+
+The following modules should be added to `devDependencies` of your
+`package.json`.
+
+- eslint
+- eslint-plugin-eslint-plugin
+- @typescript-eslint/eslint-plugin
+- @typescript-eslint/parser
```

Comment:
Can you list these additional packages on line 8 where you are showing the `npm install` command?